pandas/core/ops/__init__.py

```diff
@@ -511,7 +562,7 @@ def wrapper(self, other):
         lvalues = extract_array(self, extract_numpy=True)
         rvalues = extract_array(other, extract_numpy=True)
 
-        res_values = comparison_op(lvalues, rvalues, op)
+        res_values = comparison_op(lvalues, rvalues, op, None, {})
```

Comment:
For readability, I'd like the last two argument to be keyword arguments (It's relatively self-evident what the first three arguments for, but that's IMO not the case for the last two...).

Reply:
Will do.